app/api/utility.py

```diff
@@ -133,61 +136,74 @@ def create_query(
     str
         The SPARQL query.
     """
-    subject_level_filters = ""
+    phenotypic_session_level_filters = ""
 
     if age[0] is not None:
-        subject_level_filters += "\n" + f"FILTER (?{AGE.var} >= {age[0]})."
+        phenotypic_session_level_filters += (
+            "\n" + f"FILTER (?{AGE.var} >= {age[0]})."
+        )
     if age[1] is not None:
-        subject_level_filters += "\n" + f"FILTER (?{AGE.var} <= {age[1]})."
+        phenotypic_session_level_filters += (
+            "\n" + f"FILTER (?{AGE.var} <= {age[1]})."
+        )
 
     if sex is not None:
-        subject_level_filters += "\n" + f"FILTER (?{SEX.var} = {sex})."
+        phenotypic_session_level_filters += (
+            "\n" + f"FILTER (?{SEX.var} = {sex})."
+        )
 
     if diagnosis is not None:
-        subject_level_filters += (
+        phenotypic_session_level_filters += (
             "\n" + f"FILTER (?{DIAGNOSIS.var} = {diagnosis})."
         )
 
     if is_control is not None:
         if is_control:
-            subject_level_filters += (
+            phenotypic_session_level_filters += (
                 "\n" + f"FILTER (?{IS_CONTROL.var} = {IS_CONTROL_TERM})."
             )
         else:
-            subject_level_filters += (
+            phenotypic_session_level_filters += (
                 "\n" + f"FILTER (?{IS_CONTROL.var} != {IS_CONTROL_TERM})."
             )
 
-    if min_num_sessions is not None:
-        subject_level_filters += (
-            "\n" + f"FILTER (?num_sessions >= {min_num_sessions})."
-        )
-
     if assessment is not None:
-        subject_level_filters += (
+        phenotypic_session_level_filters += (
             "\n" + f"FILTER (?{ASSESSMENT.var} = {assessment})."
         )
 
-    session_level_filters = ""
+    subject_level_filters = ""
+    if min_num_phenotypic_sessions is not None:
+        subject_level_filters += (
+            "\n"
+            + f"FILTER (?num_phenotypic_sessions >= {min_num_phenotypic_sessions})."
+        )
+    if min_num_imaging_sessions is not None:
+        subject_level_filters += (
+            "\n"
+            + f"FILTER (?num_imaging_sessions >= {min_num_imaging_sessions})."
+        )
 
```

Comment:
should we move this section above pheno_filters now so the flow is more in line with the hierarchy (e.g. subject > pheno session - imaging session)?